app/views/content/landing/return-to-teaching-advisers/_adviser.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="row">
+  <div class="grey col-space-l">
+    <section class="col col-720 col-space-0">
+         <%= render Content::AdviserComponent.new(
+        title: "Free one-to-one support by phone, text or email",
+        intro: "If you're thinking about returning to teaching in England, an adviser can help you get classroom experience, search teaching vacancies and support you with the application process. To be eligible for an adviser, you'll need English qualified teacher status (QTS) and want to
```

Comment:
```suggestion
        intro: "If you're thinking about returning to teaching in England, an adviser can help you get classroom experience, search teaching vacancies and support you with the application process. To be eligible for an adviser, you'll need English qualified teacher status (QTS) and want to
```

this sounds a bit like advisers will help you search TV the service? Should we say `search FOR teaching vacancies` or maybe just `search for teaching jobs`?